### PR TITLE
feat(args): add multiline variable type

### PR DESCRIPTION
**Pull request by k-dovgan**
Also fix bug with 'metavar' and non-existing help in ModuleArgumentParser and deprecated 'iteritems' in github_vcs

### CODE REVIEW

File: tests/test_multiline_variables.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    assert namespace.argument == text
+
+    # File
+    tmp_file = tmp_path / "myfile.txt"
+    tmp_file.write_text(text)
+    namespace = parser.parse_args(["--argument", '@' + str(tmp_file)])
+    assert namespace.argument == text
+
+    # Stdin
+    monkeypatch.setattr('sys.stdin.read', lambda: text)
```
Comment (i-keliukh) [inline tests/test_multiline_variables.py:20]: As an alternative, you can spawn separate interpreter

File: tests/test_multiline_variables.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    text = "This is text\nwith some line breaks\n"
+    parser = ModuleArgumentParser()
+    parser.add_argument("--argument", type=multiline_argument)
+
+    # Usual value
+    namespace = parser.parse_args(["--argument", text])
+    assert namespace.argument == text
+
+    # File
+    tmp_file = tmp_path / "myfile.txt"
```
Comment (i-keliukh) [inline tests/test_multiline_variables.py:14]: I think, you can use tmpfile directly

File: universum/lib/module_arguments.py [hunk 3]
```diff
@@ -5,6 +5,15 @@
 import sys
 
 
+def multiline_argument(sting):
```
Comment (i-keliukh) [inline universum/lib/module_arguments.py:8]: sting -> string

File: tests/test_multiline_variables.py (hunk 1 above)
Comment (k-dovgan) [reply tests/test_multiline_variables.py:20]: I tried subprocess & multiprocessing, but liked this one more for better readability.

File: tests/test_multiline_variables.py (hunk 2 above)
Comment (k-dovgan) [reply tests/test_multiline_variables.py:14]: pytest doesn't have a 'tmp_file' fixture, only 'tmp_path' and 'tmp_dir', which are both directories. I can switch to built-in `tempfile.NamedTemporaryFile` if needed, but I myself like pytest's fixtures better, and also we do use them in all other tests.

File: universum/lib/module_arguments.py [hunk 4]
```diff
@@ -40,7 +39,10 @@ def _add_action(self, action):
                 action.help += "\n\nEnvironment variable: ${}".format(action.metavar)
             else:
                 if not isinstance(action, argparse._SubParsersAction):
-                    action.help += " [env: {}]".format(action.metavar)
+                    try:
+                        action.help += f" [env: {action.metavar}]"
+                    except TypeError:
+                        action.help = f"Also available as [env: {action.metavar}]"
```
Comment (i-keliukh) [inline universum/lib/module_arguments.py:45]: Please also fix sphinx case (line 39)

**Pull request merged**